File: Competitive Programming 3/Section 1.3.3/11942 - Lumberjack Sequencing/Java/Main.java
import java.util.Scanner;

public class Main {
    static Scanner s = new Scanner(System.in);
    public static void main(String[]args){
        System.out.println("Lumberjacks:");
        for(int t = s.nextInt(); t > 0; t --){
            int last=0;
            boolean negativeDif = false,positiveDif = false;
            for(int i = 0 ; i < 10 ; i ++){
                int number = s.nextInt();
                if(i>0){
                    if(number-last>0){
                        //System.out.printf("%d %d",number,last);
                        positiveDif=true;
                    }else if(number-last<0){
                        negativeDif=true;
                    }
                }
                last = number;
            }
            System.out.printf("%s\n",positiveDif^negativeDif?"Ordered":"Unordered");
        }
    }
}

File: Competitive Programming 3/Section 1.3.3/11942 - Lumberjack Sequencing/C++/main.cpp
#include<iostream>

using namespace std;

int main(){
    ios_base::sync_with_stdio(0);cin.tie(0);
    int t ; 
    cin >> t ;
    cout << "Lumberjacks:\n";
    while(t--){
        int last,number;
        bool negativeDif=false,positiveDif=false;
        cin >> last;
        for(int i = 1 ; i < 10 ; i ++){
            cin >> number;
            if(number-last>0){
                positiveDif = true;
            }else if(number-last<0){
                negativeDif = true;
            }
            last = number;
        }
        cout << (positiveDif^negativeDif?"Ordered":"Unordered") << "\n";
    }
}